src/Propel/Runtime/ActiveQuery/QueryExecutor/AbstractQueryExecutor.php

```diff
@@ -131,13 +131,16 @@ protected function executeStatement(PreparedStatementDto $preparedStatementDto)
     protected function handleStatementException(Throwable $e, ?string $sql, $stmt = null): void
     {
         $internalMessage = $e->getMessage();
-        Propel::log($internalMessage, Propel::LOG_ERR);
 
         $isDebugMode = $this->connectionIsInDebugMode();
         if ($isDebugMode && $stmt instanceof StatementWrapper) {
             $sql = $stmt->getExecutedQueryString();
         }
         $publicMessage = "Unable to execute statement [$sql]";
+
+        $fullLogMessage = $publicMessage . PHP_EOL . "Reason: [$internalMessage]";
```

Comment:
Why would you show internal messages outside of debugging mode?
They might contain sensitive data, which logged in production may bring a leak.

Why do you think it's a safe option to show?

Reply:
@PhilinTv see line 134, this log was already logged, but for some reason, the propel log half of the message in one log and half in another, for real projects it'a huge pain to understand whats going on in a such way.
I just put full log in one place, and there I added the public log to the internal one